spec/components/schemas/CheckoutPage/Taxes.yaml

```
@@ -11,4 +11,5 @@ properties:
     description: Value of tax.
     type: number
     format: float
-
+    minimum: 0
+    exclusiveMinimum: true
```

To generate correct example:
```suggestion
    exclusiveMinimum: true
    example: 0.5
```

---

Done, thanks